### PR TITLE
feat(strategies): add Ditto.money

### PR DESCRIPTION
**Pull request by developerfred**
Fixes https://github.com/Ditto-money/Bounties/issues/3.

Changes proposed in this pull request:
- add Ditto.Money Strategy 
- add Ditto.Money Space https://github.com/snapshot-labs/snapshot-spaces/pull/725/ ✅

### CODE REVIEW

**Comment by bonustrack** [comment]
@developerfred Does the test work for you? It fail for me, you can test using `npm run test --strategy=dittomoney`. It seem that the multicall `calls` param is not properly formatted. I recommend to use "Multicaller" instead, it's the new way we use to do multicall, it's much easier to build query with it, have a look on a demo here: https://github.com/snapshot-labs/snapshot.js/blob/master/test/multicaller.ts to understand how it works.

**Comment by chefchansey** [comment]
Hey guys, are any more changes required here?

File: src/strategies/dittomoney/index.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, options);
```
Comment (bonustrack) [inline src/strategies/dittomoney/index.ts:63]: You need to change to this to make it work
```suggestion
  const multi = new Multicaller(network, provider, abi, { blockTag });
```

File: src/strategies/dittomoney/examples.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      }
+    },
+    "network": "56",
+    "addresses": [
+      "0xd1a8Dd23e356B9fAE27dF5DeF9ea025A602EC81e",
+      "0x48f7cb174a2333de834452ad240ce8d3d827dc55",
+      "0x0d3cae1ff719d6a2ac0fcd7cd22f599cac64b6ec",
+      "0xE0e710A907a8E44c078e6212Efa336C1191F4CD1"
+    ],
+    "snapshot": 11605878
```
Comment (bonustrack) [inline src/strategies/dittomoney/examples.json:20]:
```suggestion
    "snapshot": 4018707
```

**Pull request merged**